src/utils/chartOptions.js

```diff
@@ -0,0 +1,284 @@
+  scales: {
+    xAxes: [{
+      display: true,
+      distribution: 'linear',
+      gridLines: {
+        display: false,
+      },
+      ticks: {
+        fontColor: chartStyles.slateGray,
+        fontSize: chartStyles.fontSize,
+        fontFamily: chartStyles.contentFontFamily,
+        maxRotation: 0,
+      },
+    }],
+
+    yAxes: [{
+      position: 'left',
+      type: 'linear',
+      gridLines: {
+        display: true,
+      },
+      ticks: {
+        display: true,
+        maxTicksLimit: 5,
+        fontColor: chartStyles.slateGray,
+        fontSize: chartStyles.fontSize,
+        fontFamily: chartStyles.contentFontFamily,
+      },
+    }],
+  },
+
+  elements: {
+    point: {
+      radius: 5,
+      hoverRadius: 4,
+      hitRadius: 10,
+    },
+    line: {
+      tension: 0,
+    },
+  },
+}, baseOptions, options);
+
+
+/**
+ * Options ONLY for Bar chart
+ * @param {object} options - More options that can be pass to the chart
+ */
+export const barChartOptions = options => merge({
+  scales: {
+    xAxes: [{
+      display: true,
+      position: 'bottom',
+      stacked: true,
+      gridLines: {
+        display: false,
+        offsetGridLines: true,
+      },
+      ticks: {
+        fontColor: chartStyles.slateGray,
+        fontSize: chartStyles.fontSize,
+        fontFamily: chartStyles.contentFontFamily,
+        maxRotation: 0,
+      },
+    }],
+
+    yAxes: [{
+      display: true,
+      position: 'left',
+      stacked: true,
+      gridLines: {
+        display: true,
+        offsetGridLines: true,
+      },
+      ticks: {
+        display: true,
+        maxTicksLimit: 5,
+        fontColor: chartStyles.slateGray,
+        fontSize: chartStyles.fontSize,
+        fontFamily: chartStyles.contentFontFamily,
+      },
+    }],
+  },
+
+  elements: {
+    rectangle: {
+      backgroundColor: chartStyles.ultramarineBlue,
+      borderWidth: '0',
+      borderColor: '',
+      borderSkipped: 'bottom',
+    },
+  },
+}, baseOptions, options);
+
+
+/**
+ * Options ONLY for Doughnut chart
+ * @param {object} options - More options that can be pass to the chart
+ */
+export const doughnutChartOptions = options => merge({
+  cutoutPercentage: 60,
+
+  elements: {
+    arc: {
+      backgroundColor: chartStyles.ultramarineBlue,
+      borderAlign: 'center',
+      borderColor: chartStyles.whiteColor,
+      borderWidth: 1,
+    },
+  },
+}, baseOptions, options);
+
+
+/**
+ * data ONLY for Line chart
+ * @param {object} data - More data that can be pass to the chart
+ */
+export const lineChartData = data => merge({
+  datasets: (data.datasets || []).map((_, index) => ({
+    backgroundColor: chartStyles.transparent,
+    borderColor: colorPallete[index],
+    pointBorderColor: colorPallete[index],
+    pointBackgroundColor: chartStyles.whiteColor,
+    pointHoverBackgroundColor: chartStyles.whiteColor,
+    pointHoverBorderColor: colorPallete[index],
+    pointHoverBorderWidth: 4,
```

Comment:
🎨 Looking at the designs, there should be no point without hover
https://projects.invisionapp.com/d/main?origin=v7#/console/18805079/391946699/preview?scrollOffset=0
https://projects.invisionapp.com/d/main#/console/17570736/374296680/preview
```suggestion
    pointHoverBorderWidth: 4,
    pointRadius: 0,
    pointHoverRadius: 4,
```